src/routes/PrimaryLayout/PrimaryLayout.js

```diff
@@ -166,7 +169,7 @@ export default class PrimaryLayout extends Component {
             <Route path='/:context(all)/:view(topics)' component={AllTopics} />
             <Route path={`/:context(all|public)/:view(map)/${OPTIONAL_POST_MATCH}`} exact component={MapExplorer} />
             <Route path={`/:context(all|public)/:view(map)/${OPTIONAL_COMMUNITY_MATCH}`} exact component={MapExplorer} />
-            <Route path={`/:context(all|public)/${OPTIONAL_POST_MATCH}`} exact component={Feed} />
+            <Route path={`/:context(all|public)/:view(stream|event|project)/${OPTIONAL_POST_MATCH}`} exact component={Feed} />
```

Comment:
will have to see/understand how this plays with the changes pospi made. and think through our routing in general